packages/compass-query-history/src/components/header/header.jsx

```diff
@@ -50,20 +55,22 @@ class Header extends PureComponent {
     const activeButton = showing === 'recent' ? 'Recent' : 'Favorites';
 
     return (
-      <div className={classnames(styles.component)}>
+      <div className={styles.component}>
         <ViewSwitcher
           dataTestId="past-queries"
           label="Past Queries"
           buttonLabels={['Recent', 'Favorites']}
           activeButton={activeButton}
-          onClick={this.onViewSwitch} />
-        <span
-          className={classnames(styles.close)}
+          onClick={this.onViewSwitch}
+        />
+        <IconButton
+          className={closeButtonStyles}
           data-test-id="query-history-button-close-panel"
-          href="#"
-          onClick={this.collapse}>
-          ×
-        </span>
+          onClick={this.collapse}
+          aria-label="Close query history"
+        >
+          <Icon glyph="X" />
```

Comment:
Update to fix the non-interactive html tag lint issue.
Before:
![Screen Shot 2022-07-06 at 3 20 52 PM](https://user-images.githubusercontent.com/1791149/177641495-2087572e-8f83-48ce-b14d-20c880564140.png)

After:
![Screen Shot 2022-07-06 at 3 22 07 PM](https://user-images.githubusercontent.com/1791149/177641529-a58fbc10-4226-42e6-8d98-fb4fbd8fdbcd.png)